### PR TITLE
Home Page

### PR DESCRIPTION
**Pull request by SolivanLau**
### Refactoring  and Keeping things DRY
- replacing background shorthand (and vars)
- leveraging global font styles, removed a lot of styles

### Addressing Tasks
- using mobile img for desktop community partnerships
- learn more links to about us in community partnerships
- removed boilerplate events
- optimized hero img
- fixing merge conflicts (wrappers)

### HTML Restructure
- worked through replacing h1 and following new `<h>`hierarchy

### CODE REVIEW

File: assets/images/pages/home/hero-home.jpg
Comment (SolivanLau) [inline assets/images/pages/home/hero-home.jpg:1]: optimized to 200kb

File: css/global.css [hunk 1]
```diff
@@ -1139,7 +1139,9 @@ Hero
 	.newsletter-form input[type="email"] {
 		width: 100%;
 	}
-
+	.newsletter-form h5 {
```
Comment (SolivanLau) [inline css/global.css:1154]: moved into global styles from **homepage.css**

File: css/home.css [hunk 2]
```diff
@@ -340,16 +322,16 @@ Community Partnerships
 ===================================
 */
 	.community {
-		background: var(--color-amaranth);
+		background-color: var(--color-amaranth);
 		padding-top: 0;
 	}
 	.community .wrapper {
-		width: 100%;
+		padding: 0;
```
Comment (SolivanLau) [inline css/home.css:379]: adjusting from global wrapper styles - using padding 0 for mobile view

**Review by maxharrisnet — CHANGES_REQUESTED** [review]
This page seems to breakdown on some tablet sizes. These screenshots show the iPad Air dimensions of 820w / 1180h. The 3-column components are too narrow and the text is getting crunched. Can you please play around with this to get things looking better on tablet?

Screen Shot 2024-01-14 at 3 59 01 AM
Screen Shot 2024-01-14 at 3 58 56 AM
Screen Shot 2024-01-14 at 3 58 43 AM

**Comment by maxharrisnet** [comment]
Looks great! We just need some adjustments to get tablet looking right and then we can merge.

**Comment by maxharrisnet** [comment]
I also notice (on tablet) that the button text breaks the line. I think we should have a smaller padding size on the left and right to prevent this and center the text. 

Screen Shot 2024-01-14 at 4 21 00 AM

File: css/home.css [hunk 3]
```diff
@@ -291,12 +291,62 @@ Media Queries (overriding styles for different screen sizes)
 			url("../assets/images/pages/home/mobile/hero-home-mobile.jpg");
 	}
 
+	/* community partnerships */
+	.community-card .text-container {
```
Comment (SolivanLau) [inline css/home.css:295]: changing the width distribution between text and img. adjusting img to fit nicer at this mq
image

File: css/home.css [hunk 4]
```diff
@@ -291,12 +291,62 @@ Media Queries (overriding styles for different screen sizes)
...
+		width: 80%;
+		padding-right: 175px;
+	}
+
+	.global-ecosystem blockquote {
+		min-width: 340px;
+		top: 130px;
+		right: 125px;
+	}
+
```
Comment (SolivanLau) [inline css/home.css:320]: Block quote has been adjusted in positioning to look smoother especially at 900px --> 768px mq

image

File: css/home.css [hunk 5]
```diff
@@ -291,12 +291,62 @@ Media Queries (overriding styles for different screen sizes)
...
 	.wwd-card {
-		width: calc(33% - var(--spacing-sm));
+		width: 100%;
+	}
+
+	.wwd-card-text {
+		min-height: auto;
+	}
+
```
Comment (SolivanLau) [inline css/home.css:334]: I played around with different widths and positions, but I feel it best to move wwd 768px adjustments to 1024px

File: css/home.css [hunk 6]
```diff
@@ -348,6 +398,10 @@ Community Partnerships
 		text-align: center;
 		padding: var(--spacing-lg) var(--spacing-sm);
 	}
+
+	.community-card a {
+		max-width: 300px;
+	}
```
Comment (SolivanLau) [inline css/home.css:404]: learn more button should have its side padding adjusted and dependent on width in tablet and mobile mqs

Comment (maxharrisnet) [reply css/home.css:404]: Much better, thanks! 🚀

File: css/home.css (hunk 5 above)
Comment (maxharrisnet) [reply css/home.css:334]: Good call

File: css/home.css (hunk 4 above)
Comment (maxharrisnet) [reply css/home.css:320]: Looks much better

File: css/home.css (hunk 3 above)
Comment (maxharrisnet) [reply css/home.css:295]: Sweet, this looks good now!

**Review by maxharrisnet — CHANGES_REQUESTED** [review]
This looks great, thank you for working out those tablet styles, refactoring, and optimizing!

**Review by maxharrisnet — APPROVED** [review]

**Pull request merged**